suite/tinytables/src/main/java/dk/alexandra/fresco/suite/tinytables/prepro/TinyTablesPreproProtocolSuite.java

```diff
@@ -43,20 +40,19 @@
 public class TinyTablesPreproProtocolSuite
     implements ProtocolSuite<TinyTablesPreproResourcePool, ProtocolBuilderBinary> {
 
+  private static final int TRIP_BATCH_SIZE = 8192;
```

Comment:
Any reason for this particular choice of batch size?

Reply:
It was the same size as before :) But the size was defined in another class before